src/Storages/System/InformationSchema/tables.sql

```diff
@@ -3,15 +3,26 @@ ATTACH VIEW tables
     `table_catalog` String,
     `table_schema` String,
     `table_name` String,
-    `table_type` Enum8('BASE TABLE' = 1, 'VIEW' = 2, 'FOREIGN TABLE' = 3, 'LOCAL TEMPORARY' = 4, 'SYSTEM VIEW' = 5),
+    `table_type` String,
+    `table_comment` Nullable(String),
+    `table_collation` Nullable(String),
     `TABLE_CATALOG` String ALIAS table_catalog,
     `TABLE_SCHEMA` String ALIAS table_schema,
     `TABLE_NAME` String ALIAS table_name,
-    `TABLE_TYPE` Enum8('BASE TABLE' = 1, 'VIEW' = 2, 'FOREIGN TABLE' = 3, 'LOCAL TEMPORARY' = 4, 'SYSTEM VIEW' = 5) ALIAS table_type
+    `TABLE_TYPE` String ALIAS table_type,
+    `TABLE_COMMENT` Nullable(String) ALIAS table_comment,
```

Comment:
Minor: Looking at [MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/information-schema-tables-table.html), I think `TABLE_COMMENT` should be listed after `TABLE_TYPE`.